Formula/rrdtool.rb

```diff
@@ -28,6 +28,11 @@ class Rrdtool < Formula
   patch :DATA
 
   def install
+    # fatal error: 'ruby/config.h' file not found
+    if MacOS.sdk && (MacOS.sdk.version != MacOS.version)
```

Comment:
👍. Maybe worth adding a comment that it is only problematic on Yosemite, but yeah, applying it unconditionally where that isn't harmful sounds good. I guess that it may well become problematic on El Cap once Xcode 8 is stable as well, sadly.